scripts/fund-dev-accounts/main.rs

```diff
@@ -0,0 +1,153 @@
+/// As Pop Network uses the relay chain token as native, the dev accounts are not funded by default.
+/// Therefore, after network launch there needs to be a reserve transfer from the relay chain
+/// to the dev accounts.
+///
+/// This script performs these reserve transfers to fund the dev accounts from the relay chain.
+///
+use subxt::{OnlineClient, PolkadotConfig};
+use subxt_signer::sr25519::{dev, Keypair};
+
+#[cfg(not(feature = "paseo"))]
+mod relay {
+	use super::*;
+	#[subxt::subxt(runtime_metadata_path = "../metadata/rococo.scale")]
+	pub(crate) mod runtime {}
+	pub(crate) type RuntimeCall = runtime::runtime_types::rococo_runtime::RuntimeCall;
+	pub(crate) const UNIT: u128 = 1_000_000_000_000;
+
+	pub(crate) use runtime::runtime_types::{
+		staging_xcm::v4::{
+			asset::Fungibility::Fungible,
+			asset::{Asset, AssetId, Assets},
+			junction::Junction,
+			junctions::Junctions,
+			junctions::Junctions::X1,
+			location::Location,
+		},
+		xcm::{v3::WeightLimit, VersionedAssets, VersionedLocation},
+	};
+
+	// generate XCM message to reserve transfer funds to a designated account on
+	// Pop Parachain
+	pub(crate) fn gen_account_fund_message_call(account: Keypair) -> RuntimeCall {
+		let pop_location = VersionedLocation::V4(Location {
+			parents: 0,
+			interior: X1([Junction::Parachain(9090)]),
```

Comment:
We might want to use the paraId set in the chainspec for pop network on rococo local here https://github.com/r0gue-io/pop-node/pull/46

`4385`

Changing the one in the zombienet file would be nice as well.